loginpass/vk.py

```diff
@@ -0,0 +1,105 @@
+"""
+    loginpass.vk
+    ~~~~~~~~~~~~~~~~~~~
+
+    Authenticate using `VK <https://vk.com>`_.
+
+    Useful links:
+
+    - Register an application: https://vk.com/editapp?act=create
+    - API documentation: https://vk.com/dev/users.get
+
+    :copyright: (c) 2018 by Hsiaoming Yang
+    :license: AGPLv3+, see LICENSE for more details.
+"""
+
+import datetime
+from authlib.specs.oidc import UserInfo
+from authlib.common.urls import add_params_to_uri
+from ._core import OAuthBackend, map_profile_fields
+
+
+def vk_compliance_fix(session):
+    def _add_extra_info(url, headers, body):
+        params = {'v': '5.80', 'fields': 'sex,bdate,has_photo,photo_max_orig,site,screen_name'}
+        url = add_params_to_uri(url, params)
+        return url, headers, body
+
+    def _token_response(resp):
+        token = resp.json()
+        token['token_type'] = 'Bearer'
+        resp.json = lambda: token
+        return resp
+
+    session.register_compliance_hook('protected_request', _add_extra_info)
```

Comment:
This seems not right. These params are used only for `profile` I guess. Why not add them to `users.get`?